draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -202,8 +199,8 @@ e
 r
 : The prime order of the G1 and G2 subgroups.
 
-P1, P2
-: points on G1 and G2 respectively. For a pairing-friendly curve, this document denotes operations in E1 and E2 in additive notation, i.e., P + Q denotes point addition and x \* P denotes scalar multiplication. Operations in GT are written in multiplicative notation, i.e., a \* b is field multiplication.
+BP, BP'
```

Comment:
We don't need BP (the G1 base point), so we can get rid of it.

Reply:
`BP` (renamed to `BP1`) is used in the description of `P1` in the ciphersuite (i.e., that `P1 != BP1`). 

Still could most likely remove it, but better be on the safe side??